test/parallel/test-fs-read-stream-fd-leak.js

```diff
@@ -29,20 +29,20 @@ function testLeak(endFn, callback) {
   var i = 0;
   var check = 0;
 
-  var checkFunction = function() {
-    if (openCount != 0 && check < totalCheck) {
+  const checkFunction = function() {
+    if (openCount !== 0 && check < totalCheck) {
       check++;
       setTimeout(checkFunction, 100);
       return;
     }
-    assert.equal(0, openCount, 'no leaked file descriptors using ' +
+    assert.strictEqual(0, openCount, 'no leaked file descriptors using ' +
                  endFn + '() (got ' + openCount + ')');
```

Comment:
Template string was a great idea. I've followed up with a commit - how do you like the look of it?